janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/builder/CQLStoreFeaturesBuilder.java

```diff
@@ -62,6 +63,10 @@ public CQLStoreFeaturesWrapper build(final CqlSession session, final Configurati
         fb.optimisticLocking(true);
         fb.multiQuery(false);
 
+        if (!configuration.get(TTL_ENABLED)) {
+            fb.cellTTL(false).storeTTL(false);
```

Comment:
I didn't check that but I see `fb.timestamps(false);` was removed below and was wandering if TTL is disabled shoudn't we set timestamps to `false`?
I.e. I would assume something like below but could be wrong
```
if(configuration.get(TTL_ENABLED)){
  fb.cellTTL(true).timestamps(true);
} else {
  fb.cellTTL(false).storeTTL(false).timestamps(false);
}
```
That was my assumption but I really didn't check how they are used

Reply:
If you think we should always have `timestamps(true)` no matter TTL is enabled or not then just ignore my comment :)

Reply:
No, `timestamps(true)` option has nothing to do with TTL. Honestly, I am not very sure what that option was meant for, but I have played around with `timestamps(false)` with and without TTL for a while and didn't see any problem. 

I don't really remember why I put `timestamps(false)` in the first place. It must be because I somehow thought `timestamps` and `ttl` options are the same. Apparently, they are not.

Reply:
`timestamps(false)` was introduced by this PR: https://github.com/JanusGraph/janusgraph/pull/2644/files#diff-e7acaf1d98c639f210cd458192411d7e42bee5900aed88d7fc90d5909781c3baR235